### ISSUE TITLE
Dry run errors, i.e. SU2_CFD -d <configname>.cfg

### ISSUE THREAD

**Issue by TobiKattmann** [su2code/SU2#857]
**Describe the bug**
When running any dry run in the form
`SU2_CFD -d <configname>.cfg`
an integer divide by zero (`Floating point exception: integer divide by zero`) happens.

A backtrace pointed me towards `roundUpDiv (numerator=0, denominator=0)` in `omp_structure.hpp` +104 and further up `computeStaticChunkSize (totalWork=0, numThreads=1, maxChunkSize=4096)`. I guess one might check somewhere along the line whether there is even work to do, i.e.  wrap the `*.Initialize(nPoint,...)` in `C<Inc>EulerSolver` etc in `if (nPoint != 0)` conditionals (that got the Quickstart working again for me). But maybe there is a better more elegant solution without too much overhead.

I would like @pcarruscag to take a look as the omp stuff is involved :)

**To Reproduce**
Every dryrun I tried, using Quickstart should be enough to understand the problem.

**Desktop (please complete the following information):**
 - OS: Red Hat Enterprise Linux 7.6
 - C++ compiler and version: GCC 4.8.5
 - MPI implementation and version: openMPI 3.1.2
 - SU2 Version: latest develop, but this probably affects 7.0.1 as well, i.e. hotfix time 🕙

**Comment by pcarruscag** [comment]
It is probably just the linear algebra classes being initialized with size 0.

**Comment by LorenzSLH** [comment]
I would like to reopen the issue regarding the dry run. When having turbomachine settings on, the dry run fails when calculating (preprocessing the turbomachine geometry) with the following error:

- Case Testcases APU_Turbomachinery
- Version 7.5.1 
- Command: SU2_CFD -d Jones_restart.cfg

```
---------------------- Turbomachinery Preprocessing ---------------------
Initialize Turbo Vertex Structure.
Number of span-wise sections in Zone 0: 0.
Number of span-wise sections in Zone 1: 0.
Max number of span-wise sections among all zones: 0.
Initialize solver containers for average and performance quantities.
Compute inflow and outflow average geometric quantities.
Set span-wise sections between zones on Mixing-Plane interface.
Transfer average geometric quantities to zone 0.
[---] *** Process received signal ***
[---]  Signal: Segmentation fault (11)
[---]  Signal code: Address not mapped (1)
[---]  Failing at address: 0x8
[---]  [ 0] /lib/x86_64-linux-gnu/libc.so.6(+0x43090)[0x7ffa2d583090]
[---]  [ 1] SU2_CFD(+0x5e6dc4)[0x55c98b9dedc4]
[---]  [ 2] SU2_CFD(+0x1ab51e)[0x55c98b5a351e]
[---]  [ 3] SU2_CFD(+0x1bc2d3)[0x55c98b5b42d3]
[---]  [ 4] SU2_CFD(+0x201b26)[0x55c98b5f9b26]
[---]  [ 5] SU2_CFD(+0x151f79)[0x55c98b549f79]
[---]  [ 6] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3)[0x7ffa2d564083]
[---]  [ 7] SU2_CFD(+0x171b3e)[0x55c98b569b3e]
[---]  *** End of error message ***
Segmentation fault
```

i guess in the dry run mode, some algorithm are ignored and therefore the segmentation fails. 
I was wondering: 

1) Are there any additional output variables that come with the turbomachinery settings (besides those, available in the folder) 

2) If not, available output for the solver can be achieved by just dry running a case that has the same solver but no turbomachine settings on. Maybe a flag in the dry run option could be implemented on, that deactives the turbomachine settings in the case, to avoid the error message.

**Comment by pcarruscag** [comment]
There will be new outputs after #2011 
Those places where the code is segfaulting need to be bypassed when the geometry is empty, should not be a big deal.
@joshkellyjak do you think you can recruit someone from the turbo group to help out?

**Comment by joshkellyjak** [comment]
Will look into it, there's a known issue fixed in #2011 that also throws a segfault where the above error occurs. It could already be fixed, if not I will fix.

**Comment by joshkellyjak** [comment]
Update: This should be fixed in #2011, apologies this took so long to get round it in what is a very simple fix.
